tests/schema/test_base.py

```diff
@@ -95,3 +95,20 @@ def test_dunder_name() -> None:
 
 def test_dunder_name_with_rule() -> None:
     assert MySchemaWithRule.__name__ == "MySchemaWithRule"
+
+
+def test_user_error_tuple_column() -> None:
```

Comment:
I think we are currently still missing a test that shows that the user is allowed to specify extra members (i.e. having a valid, non-column member in a Schema). Could you add one?

Reply:
probably a tuple of something other than `dy.Column` so that we know that our check above works correctly